docs/user/agent/add-node/add-nodes.md

```diff
@@ -84,7 +84,8 @@ $ ./node-joiner.sh config.yaml
 Use the iso image to boot all the nodes listed in the configuration file, and wait for the related
 certificate signing requests (CSRs) to appear. When adding a new node to the cluster, two pending CSRs will
 be generated, and they must be manually approved by the user.
-Use the following command to monitor the pending certificates:
+
+Use the following command or [node-joiner-monitor.sh](./node-joiner-monitor.sh) described below to monitor the pending certificates:
```

Comment:
Just noticed that below there's a small pre-existing typo, `User` instead of `use`